src/java.base/share/classes/java/nio/charset/spi/CharsetProvider.java

```diff
@@ -33,17 +33,29 @@
  * Charset service-provider class.
  *
  * <p> A charset provider is a concrete subclass of this class that has a
- * zero-argument constructor and some number of associated charset
- * implementation classes.  Charset providers may be installed in an instance
- * of the Java platform as extensions.  Providers may also be made available by
- * adding them to the application class path or by some other
- * platform-specific means.  Charset providers are looked up via the current
- * thread's {@link java.lang.Thread#getContextClassLoader() context class
- * loader}.
+ * zero-argument constructor and some number of associated {@code Charset}
+ * implementation classes.  Charset providers are deployed by adding them to either
+ * the application module path or the application class path. In order to be looked
```

Comment:
The latest update to the class description looks quite good. I think this sentence can be simplified down to "Charset provider are deployed on the application module path ...".

Reply:
Fixed in https://github.com/openjdk/jdk/pull/21076/commits/08e8161c3d9a6c2efca7b5f578a72060537495e9

Reply:
Thanks for the approval @AlanBateman . Could you also review the [CSR](https://bugs.openjdk.org/browse/JDK-8340412) when you get the chance.